src/Honeybee_Add to EnergyPlus Library.py

```diff
@@ -50,7 +50,32 @@
 
 import scriptcontext as sc
 import Grasshopper.Kernel as gh
+import os
 
+def prepEPObjects(rawStrings):
```

Comment:
Ah! I forgot to copy the link. [Here it is!](https://github.com/mostaphaRoudsari/honeybee/blob/master/src/Honeybee_Honeybee.py#L568-L581)
Yes. The idf file. I have seen several cases that people do stuff that is fine with EP but will have issues for us to import. This method is trying to address that.